docs/maintainers/growing-your-project.md

```diff
@@ -18,33 +18,36 @@ When it comes to deciding on whether your open source project is ready to grow,
+Like Superman, Wonder Woman, the Flash, and Batman in ***The Justice League***, the people you bring on your team(s) and your project's contributors are key to its success, which is why picking the right people is important.  When doing this process, consider going to their OpenSauced profiles and checking their [dev card](../docs/features/dev-card.md). While you do this, ask yourself the following questions:
 
 - How often do they contribute to yours and other open source projects in general?
 - Are they invested in your project's success?
 
-Once you have your list of potential candidates, [create an Insights page for your project](../docs/maintainers-guide.md#creating-a-new-repository-insight-page), and add your candidates. From there, monitor their progress by reviewing the contributions that they post in the Highlights section. As you analyze this information, conisder using the following questions to help with this step in your open source project's growth journey:
+Once you have your list of potential candidates, [create an Insights page for your project](../docs/maintainers-guide.md#creating-a-new-repository-insight-page), and add your candidates. From there, monitor their progress by reviewing the contributions that they post in the Highlights section. As you analyze this information, consider using the following questions to help with this step in your open source project's growth journey:
 
 - How have they grown since you found their profile?
 - What types of contributions does this person frequently make to yours and other open source projects?
 From there, consider narrowing down your list to If you need more tips on how to pick people that can help grow your project, check out our blog post, ["How to Build Your Open Source Dream Team"](https://dev.to/opensauced/how-to-build-your-open-source-dream-team-a-guide-3i90).
 
 ### Recruiting team members and contributors
 
-When you're ready to recruit your team members, consider going to their OpenSauced profiles and send them a message via the contact information they provide. If you and they have a PRO account, you can also send them invites using the [connections feature](../../docs/features-connections.md). 
+When you're ready to recruit your team members, consider creating an Insights page with projects that are similar to yours. From there, determine which contributors you think suits your project, add them to a list, and monitor their progress.  Once you've finished your observations, recruit them. You can do this by going to their OpenSauced profiles and sending them a message via the contact information they provide.
 
-To recruit new contributors, consider doing the following methods:
+:::note If both of you have a PRO account, you can also send them invites using the [connections feature](../../docs/features-connections.md).
```

Comment:
This is not true

Reply:
We omitted connection from [our docs](https://docs.opensauced.pizza/). So, I don't think we should even mention connections anywhere in the docs.